### ISSUE TITLE
Panic in a query

### ISSUE THREAD

**Issue by awegrzyn** [influxdata/platform#140]
`./influx repl --storage-hosts "localhost1:8082","localhost2:8082"`:

```
> from(db:"udp")
  |> filter(fn: (r) => r._measurement == "readoutRate" AND r._field == "value")
  |> range(start:-10s)
{id: "range25", kind: "range"}
> 
> run()
Result: _result
Error: panic: runtime error: invalid memory address or nil pointer dereference
goroutine 99 [running]:
runtime/debug.Stack(0xb6d620, 0xb64800, 0x112ce80)
	/usr/local/go/src/runtime/debug/stack.go:24 +0xa7
github.com/influxdata/platform/query/execute.(*executionState).do.func1.1(0xc42023a500)
	/go/src/github.com/influxdata/platform/query/execute/executor.go:186 +0xb9
panic(0xb64800, 0x112ce80)
	/usr/local/go/src/runtime/panic.go:502 +0x229
github.com/influxdata/platform/query/execute.partitionKeyLess(0xd00dc0, 0xc42054a360, 0x0, 0x0, 0x0)
	/go/src/github.com/influxdata/platform/query/execute/partition_key.go:117 +0x68
github.com/influxdata/platform/query/execute.(*partitionKey).Less(0xc42054a360, 0x0, 0x0, 0xc91868)
	/go/src/github.com/influxdata/platform/query/execute/partition_key.go:59 +0x4b
github.com/influxdata/platform/query/functions/storage/pb.(*mergedStreams).determineNewKey(0xc4201c7e60, 0x0)
	/go/src/github.com/influxdata/platform/query/functions/storage/pb/reader.go:905 +0xe8
github.com/influxdata/platform/query/functions/storage/pb.(*mergedStreams).more(0xc4201c7e60, 0xbc4f40)
	/go/src/github.com/influxdata/platform/query/functions/storage/pb/reader.go:875 +0x114
github.com/influxdata/platform/query/functions/storage/pb.(*bockIterator).handleRead(0xc42004b000, 0xc4204dc7d0, 0xc4201c7e60, 0xc420405180, 0xcf9880)
	/go/src/github.com/influxdata/platform/query/functions/storage/pb/reader.go:150 +0x60
github.com/influxdata/platform/query/functions/storage/pb.(*bockIterator).Do(0xc42004b000, 0xc4204dc7d0, 0xc4203f9c80, 0xc4201c7c20)
	/go/src/github.com/influxdata/platform/query/functions/storage/pb/reader.go:146 +0x64f
github.com/influxdata/platform/query/functions/storage.(*source).run(0xc4201ff9e0, 0xcf67c0, 0xc4203f9c80, 0x0, 0xc4206c7f80)
	/go/src/github.com/influxdata/platform/query/functions/storage/storage.go:99 +0x1ec
github.com/influxdata/platform/query/functions/storage.(*source).Run(0xc4201ff9e0, 0xcf67c0, 0xc4203f9c80)
	/go/src/github.com/influxdata/platform/query/functions/storage/storage.go:83 +0x43
github.com/influxdata/platform/query/execute.(*executionState).do.func1(0xc42023a500, 0xcf67c0, 0xc4203f9c80, 0xcefc20, 0xc4201ff9e0)
	/go/src/github.com/influxdata/platform/query/execute/executor.go:189 +0x6b
created by github.com/influxdata/platform/query/execute.(*executionState).do
	/go/src/github.com/influxdata/platform/query/execute/executor.go:174 +0x83
```

**Comment by awegrzyn** [comment]
Actually `./influx repl --storage-hosts "localhost2:8082"`
gives
```dial tcp 10.163.38.11:8082: connect: no route to host```

**Comment by nathanielc** [comment]
This query panics when specifying multiple storage hosts.

**Comment by nathanielc** [comment]
Fixed by #141
